base/_base.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
```

Comment:
Yeah, I'm ok with that. Some instances ca for " ' ' " nested quotes, and it's so conditional that I'm ok killing that check